cluster-autoscaler/simulator/predicatechecker/schedulerbased.go

```diff
@@ -71,7 +72,8 @@ func NewSchedulerBasedPredicateChecker(informerFactory informers.SharedInformerF
 	}
 
 	checker := &SchedulerBasedPredicateChecker{
-		framework:              framework,
+		profiles:               profiles,
+		defaultSchedulerName:   schedConfig.Profiles[0].SchedulerName,
```

Comment:
Is it a requirement for the first config in file to be the default one? If not this seems very brittle.

Reply:
use profiles[0] as default, check len in line 59.